storybookjs/storybook
# added react Error overlay on cli template by default

closes issue #1088

Issue:

## What I did
formalized the steps outlined in https://github.com/storybooks/storybook/issues/2890 adding it to the middleware and in the react CLI

now it looks like this when you throw an error in the react storybook ![woo](https://user-images.githubusercontent.com/35976578/35652079-21c097fe-06b0-11e8-8547-99f01abaccfc.png)

## How to test
just startup a new storybook project

Is this testable with jest or storyshots? not sure

Does this need a new example in the kitchen sink apps? not sure

Does this need an update to the documentation? probably not

If your answer is yes to any of these, please make sure to include it in your PR.
